packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -145,8 +144,145 @@ class InteractiveViewer extends StatefulWidget {
+  /// for [Stack], [CustomMultiChildLayout], or any widget that can position children irregardless
+  /// if the parent is an InteractiveViewer.
+  ///
+  /// This is fine for non-interactive elements and in most cases using [InteractiveViewer] or [InteractiveViewer.builder] will be
+  /// the desired behavior.
+  ///
+  /// However if you need hit tests for all transformed children you can use [InteractiveViewer.raw] and
+  /// handle child transforms manually.
+  ///
+  /// With the following example the child will never exceed the bounds of the parent and always stay static, preserving hit tests.
+  ///
+  /// ```dart
+  /// import 'package:flutter/widgets.dart';
+  /// import 'package:vector_math/vector_math_64.dart' show Matrix4, Quad;
+  ///
+  /// typedef Node = ({Rect rect, Widget child});
+  ///
+  /// class Example extends StatelessWidget {
+  ///   const Example({
+  ///     super.key,
+  ///     required this.controller,
+  ///     required this.nodes,
+  ///   });
+  ///
+  ///   final TransformationController controller;
+  ///   final List<Node> nodes;
+  ///
+  ///   @override
+  ///   Widget build(BuildContext context) {
+  ///     return InteractiveViewer.raw(
+  ///       transformationController: controller,
+  ///       builder: (BuildContext context, Quad viewport) {
+  ///         final Matrix4 matrix = controller.value;
+  ///         return Stack(
+  ///           clipBehavior: Clip.none,
+  ///           children: nodes.map((Node node) {
+  ///             final Rect rect = MatrixUtils.transformRect(matrix, node.rect);
+  ///             return Positioned.fromRect(
+  ///               rect: rect,
+  ///               child: GestureDetector(
+  ///                 onTap: () => print('Node clicked'),
+  ///                 child: node.child,
+  ///               ),
+  ///             );
+  ///           }).toList(),
+  ///         );
+  ///       },
+  ///     );
+  ///   }
+  /// }
+  /// ```
+  InteractiveViewer.raw({
+    super.key,
+    this.panAxis = PanAxis.free,
+    this.boundaryMargin = EdgeInsets.zero,
+    // These default scale values were eyeballed as reasonable limits for common
+    // use cases.
+    this.maxScale = 2.5,
+    this.minScale = 0.8,
+    this.interactionEndFrictionCoefficient = _kDrag,
+    this.onInteractionEnd,
+    this.onInteractionStart,
+    this.onInteractionUpdate,
+    this.panEnabled = true,
+    this.scaleEnabled = true,
+    this.scaleFactor = 200.0,
+    this.transformationController,
+    this.trackpadScrollCausesScale = false,
+    required InteractiveViewerWidgetBuilder this.builder,
+  }) : assert(minScale > 0),
+       assert(interactionEndFrictionCoefficient > 0),
+       assert(minScale.isFinite),
+       assert(maxScale > 0),
+       assert(!maxScale.isNaN),
+       assert(maxScale >= minScale),
+       // boundaryMargin must be either fully infinite or fully finite, but not
```

Comment:
There's a missing "The" here

Reply:
It was missing for the other constructors but adding it now to all 3